### PR TITLE
Add MSI DSL to query under the gems directory

### PR DESCRIPTION
**Pull request by ksubrama**

### CODE REVIEW

File: lib/omnibus/packagers/msi.rb [hunk 1]
```diff
@@ -312,6 +314,38 @@ def signing_identity(thumbprint= NULL, params = NULL)
...
+    #
+    # @param [String] glob
+    #   a ruby acceptable glob path such as with **, *, [] etc.
+    #
+    # @return [String] path relative to the project's install_dir
+    #
+    # Raises exception the glob matches 0 or more than 1 file/directory.
+    #
+    def gem_path(glob = NULL)
+
```
Comment (chefsalim) [inline lib/omnibus/packagers/msi.rb]: remove whitespace line to match formatting

**Comment by chefsalim** [comment]
:+1: other than the whitespace

File: lib/omnibus/packagers/msi.rb [hunk 2]
```diff
@@ -312,6 +314,37 @@ def signing_identity(thumbprint= NULL, params = NULL)
     expose :signing_identity
 
     #
+    # Discovers a path to a gem/file included in a gem under the install directory.
+    #
+    # @example
+    #   gem_glob 'chef-[0-9]*-mingw32' -> 'some/path/to/gems/chef-version-mingw32'
```
Comment (schisamo) [inline lib/omnibus/packagers/msi.rb]: I think this should be `gem_path`.

**Comment by schisamo** [comment]
Other than my small comment looks :+1:

**Pull request merged**